bcda/models/service.go

```diff
@@ -167,13 +167,13 @@ func (s *service) CancelJob(ctx context.Context, jobID uint) (uint, error) {
 		job.Status = JobStatusCancelled
```

Comment:
Can we put a comment saying that the CancelJob assumes that the job exists for the supplied ID.

Reply:
Updated comment to call out CancelJob assumes the job exists.